src/historyManager.js

```diff
@@ -1,11 +1,11 @@
 /**
  * @typedef HistoryManager
- * @description Core functions to handle the update of editorjs blocks based on history log
- * @property {Object} editor — Editor.js instance object.
- */
+ * @description Core functions for handling the update of editor blocks based on the history log
```

Comment:
Can we move this **description** after the properties, as the other functions for consistency?